cluster/up.sh

```diff
@@ -21,37 +21,25 @@ kubevirtci::install
 
 $(kubevirtci::path)/cluster-up/up.sh
 
-# Deploy CNA
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/namespace.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/network-addons-config.crd.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/operator.yaml
-./cluster/kubectl.sh create -f ./hack/cna/cna-cr.yaml
-
-# wait for cluster operator
+echo 'Deploying Linux bridge CNI and Multus ...'
+./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.25.0/namespace.yaml
+./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.25.0/network-addons-config.crd.yaml
+./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.25.0/operator.yaml
+cat <<EOF | ./cluster/kubectl.sh create -f -
```

Comment:
I noticed that you generally change from using yaml files to heredocs, could you explain why you prefer this?

Reply:
Less jumps between files. Once we drop dind, all the dependencies would be here. But if you prefer it being in a separate file, I don't mind that much.